### ISSUE TITLE
[EN][Item] Quest item respawn times

### ISSUE THREAD

**Issue by saduss** [Atlantiss/BugTracker#5798]
Quests items that needs to be picked up have long respawn times (5-10 minutes). Moreover, quests that required players to destroy objects have long respawn times as well. This makes group questing extremely frustrating since it slows down questing and waste our time by waiting 20+ minutes just for each party member to loot an item. Thanks in advance.

Example: Conpendium of the fallen (http://www.wowhead.com/quest=26993/compendium-of-the-fallen)

**Comment by Tulv** [comment]
Yes, please prioritise this to be fixed.  Very annoying to have to wait for each person in your group to get the quest completed.
